test/controllers/msb/SendMoneyToOtherCountryControllerSpec.scala

```diff
@@ -136,6 +137,57 @@ class SendMoneyToOtherCountryControllerSpec extends AmlsSpec with MockitoSugar {
       redirectLocation(result) must be(Some(controllers.msb.routes.SendTheLargestAmountsOfMoneyController.get().url))
     }
 
+    "on valid post where the value is false and was previously true" in new Fixture {
+
+      val newRequest = request.withFormUrlEncodedBody(
+        "money" -> "false"
+      )
+      val msbServices = Some(BusinessMatchingMsbServices(
+        Set(
+          TransmittingMoney,
```

Comment:
I think this test should probably use `val msbServices = Some(BusinessMatchingMsbServices(Set()))`, considering the context of the test directly above.
This test would probably go further down in the file.

Reply:
Or at least without CurrencyExchange?